edx_proctoring/static/proctoring/js/views/proctored_exam_add_allowance_view.js

```diff
@@ -155,6 +162,47 @@ var edx = edx || {};
                 });
             }
         },
+        selectExamAtIndex: function (index) {
+            var selectedExam = this.proctored_exams[index];
+
+            if (selectedExam.is_proctored) {
+                if (selectedExam.is_practice_exam)
+                    $('#exam_type_label').text(gettext('Practice Exam'));
+                else
+                    $('#exam_type_label').text(gettext('Proctored Exam'));
+
+                $('#allowance_type').show();
+                $('#timed_exam_allowance_type').hide();
+                $('#allowance_type').val(this.allowance_types[0][0]);
+                this.selectAllowanceLabelsText(this.allowance_types[0][0]);
+            }
+            else {
+                $('#exam_type_label').text(gettext('Timed Exam'));
+
+                // Even though we have the Select element hidden, the backend will still be using
+                // the Select's value for the allowance type (key).
+                $('#allowance_type').val('additional_time_granted').hide();
+
+                $('#timed_exam_allowance_type, #timed_exam_mins_label').show();
+                $('#allowance_value_label').text(gettext('Additional Time'));
```

Comment:
Can we replace the above two lines with the following?
```
this.selectAllowanceLabelsText('additional_time_granted');
```
And move the '#timed_exam_allowance_type` toggle into the `selectAllowanceLabelsText` function?